webpack.config.js

```diff
@@ -79,3 +79,10 @@ if ( 'production' === process.env.NODE_ENV ) {
 } else {
 	config.devtool = 'source-map';
 }
+
+if ( 'test' === process.env.NODE_ENV ) {
+	config.target = 'node';
+	config.externals = [ nodeExternals() ];
```

Comment:
Not sure what the overhead is like for requiring a module, probably not much. But an option may be to inline it as:

```js
config.externals = [ require( 'webpack-node-externals' )() ];
```

I'm not too concerned about it.